src/libraries/System.Globalization/tests/System/Globalization/TextInfoTests.cs

```diff
@@ -368,6 +369,7 @@ public static IEnumerable<object[]> ToUpper_TestData()
             }
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/36672", TestPlatforms.Android)]
```

Comment:
Same as `TestToLower`.  Apply this attribute to tests that call into this method.

Reply:
Fixed my script to put active issues above the test function definition, ignoring any calls of the test function